internal/codespaces/api/api.go

```diff
@@ -394,6 +394,56 @@ func (a *API) GetCodespace(ctx context.Context, codespaceName string, includeCon
 	return &response, nil
 }
 
+func (a *API) GetOrgMemberCodespace(ctx context.Context, orgName string, userName string, codespaceName string) (*Codespace, error) {
+	perPage := 100
+	listURL := fmt.Sprintf("%s/orgs/%s/codespaces?per_page=%d", a.githubAPI, orgName, perPage)
```

Comment:
Should this be `/orgs/%s/members/%s/codespaces`? Otherwise this function ignores `userName`

Reply:
Fixed in https://github.com/cli/cli/pull/5812